static/app/utils/events.tsx

```diff
@@ -228,12 +228,34 @@ function hasTrace(event: Event) {
  * by ensuring that every inApp frame has a valid sourcemap
  */
 export function eventHasSourceMaps(event: Event) {
-  const inAppFrames = getInAppFrames(event);
+  const inAppFrames = getFrames(event, true);
 
   // the map field tells us if it's sourcemapped
   return inAppFrames.every(frame => !!frame.map);
 }
 
+/**
+ * Function to determine if an event has been symbolicated
+ * (rawStacktrace is backfilled by stacktrace processors after processing)
+ */
```

Comment:
Do we actually wanna track if the event was successfully symbolicated as well?
`raw_stacktrace` can be backfilled but maybe all frames are either `missing`, `unknown_image`, etc.
Maybe checking if at least 1 or more is `symbolicated` makes sense?

Reply:
We could do that, but unfortunately for java/android events we do not set these properties, because they don't go through symbolicator as of now. 

But I guess we can just check for presence of `symbolicator_status` - if it's there, we measure success based on its value, if it's not there we automatically assume that it was successful. How does that sound?

Alternatively we could introduce a new field for tracking symbolicator status and keep `is_symbolicated` as is, but you can combine those two if you want to build analytics around successfully symbolicated events

Reply:
I guess it depends on what we want to track here.
If the goal was: if the event was successfully symbolicated, at least one frame has to be `symbolicated` (when this property is backfilled).
I guess every event that requires symbolication has this field backfilled, the question is if it's symbolicated or not, isn't it?

Reply:
not sure if the check for at least one frame is sufficient though, because system frames will be most likely always symbolicated, right? Don't we provide some system images ootb?

Reply:
we run https://github.com/getsentry/symbol-collector yes, but its a best effort, can[t tell if 1 is enough, good point, maybe we have to check if there's at least 1 symbolicated `and` it is `inApp`, if possible.

Reply:
Adapted in https://github.com/getsentry/sentry/pull/50139/commits/fc384e460dbe50673337c31d6cf9cbf8d1960e2a